### PR TITLE
Move remaining enum arrays to bodypart

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary

SUMMARY: Infrastructure "Move remaining enum arrays to bodypart"

#### Purpose of change
Part of #39068

#### Describe the solution

- [x] body_wetness 
- [x] drench_capacity
- [x] temp_cur
- [x] frostbite_timer
- [x] temp_conv
#### Testing

Load Sweet Cataclysm
Start as a Sugar Boy
Jump in a river
Dissolve in the water

Reload a save > No Pb
Save and reload > No pb
Get naked and change season to winter > start to freeze
Get closth back back to spring adn go to basement > temperature goes back up
New game>No immediate problem

### CODE REVIEW

File: src/bodypart.cpp [hunk 1]
```diff
@@ -400,6 +402,11 @@ bool bodypart::is_at_max_hp() const
     return hp_cur == hp_max;
 }
 
+float bodypart::get_wetness_percentage() const
+{
+    return static_cast<float>( wetness / drench_capacity );
```
Comment (BevapDin) [inline src/bodypart.cpp:407]:
```suggestion
    return static_cast<float>( wetness ) / drench_capacity;
```

**Comment by kevingranade** [comment]
The blood tracking PR conflicts with this.

File: src/savegame_json.cpp [hunk 2]
```diff
@@ -543,20 +543,7 @@ void Character::load( const JsonObject &data )
...
-    temp_conv.fill( 5000 );
-    data.read( "temp_conv", temp_conv );
-
-    frostbite_timer.fill( 0 );
-    data.read( "frostbite_timer", frostbite_timer );
-
-    body_wetness.fill( 0 );
-    data.read( "body_wetness", body_wetness );
-
+
```
Comment (kevingranade) [inline src/savegame_json.cpp:546]: Darn whitespace issues.
```suggestion

```

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/my-torso-and-arms-warmth-drops-to-insane-levels-after-loading-a-save/24201/2

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/my-torso-and-arms-warmth-drops-to-insane-levels-after-loading-a-save/24201/4

**Pull request merged**